### ISSUE TITLE
Option to disable popups - or move them somewhere else.

### ISSUE THREAD

**Issue by klement** [zanllp/sd-webui-infinite-image-browsing#524]
Hi and thanks for a great tool!

I have a UI suggestion:

When using shortcuts to delete or like images while browsing in full screen, the popups are - in my opinion - in almost the worst place - over top of image (the absolute worst place would be centre of screen). I would appreciate a way to either turn these off, move them to a corner or make them disappear when moving to a new image.

I'm all in for action feedback, but not if it's getting in the way. Popups linger too long and when sifting through say portrait images, they are rendered over faces. I think having to wait until the popup goes away is a property that could be improved. I think it'd be best if the popup relating to a particular image goes away when that image goes away. So when an image is deleted, the popup is very short. When an image is liked and then an arrow is pressed, the popup should be forcibly dismissed when a different image is shown.

**Comment by zanllp** [comment]
Are you referring to this thing?   

<img width="754" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/3863c3ed-61c5-4c19-83d9-80ed30051b52">

**Comment by klement** [comment]
Yes, exactly those popups.

**Comment by PoisonJam7** [comment]
Agreed. They're extremely annoying. Sometimes they disappear quickly, but other times they linger for far too long. It would be nice if we could move them or disable them entirely.
